include/EASTL/internal/type_compound.h

```diff
@@ -710,6 +710,12 @@ namespace eastl
 
 	#endif
 
+	#ifdef EASTL_COMPILER_INTRINSIC_TYPE_TRAITS_AVAILABLE
+		// is_final requires compiler support detect final classes: __is_final
+		template <typename T>
+		struct is_final : public integral_constant<bool, __is_final(T)> {};
```

Comment:
We should provide a default implementation so that use code doesn't need to create seperate code paths.  If the compiler instrinic isn't available have is_final return false_type.

Reply:
Sure, that makes sense.